lib/ansible/modules/cloud/amazon/emr_cluster_find.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: emr_cluster_find
+short_description:  Find AWS Elastic Map Reduce
+version_added: "2.6"
+description:
+  - return a list of Cluster EMR based on name and state
+  - results can be sorted
+  - It depends on boto3
+options:
+  region:
+    description:
+      - region where this module will find EMR
+  state:
```

Comment:
Since `state` is used in Ansible modules as the state ansible should leave a resource in, renaming this to something like `status` or `emr_status` may be less confusing.